app-backend/db/src/main/scala/com/azavea/rf/database/SceneDao.scala

```diff
@@ -147,7 +148,27 @@ object SceneDao extends Dao[Scene] with LazyLogging {
       _ <- bandInsert
       // It's fine to do this unsafely, since we know we the prior insert succeeded
       sceneWithRelated <- SceneWithRelatedDao.unsafeGetScene(sceneId)
-    } yield sceneWithRelated
+      // we should kickoff ingests if users create scenes with any ingest status other than ingested
+      // since status fields are required, and the link between ingest status and being able to view tiles
+      // is extremely not obvious, as we've learned several times
+      kickoffIngest = sceneWithRelated.statusFields.ingestStatus == IngestStatus.Queued
```

Comment:
This does not seem to express the comments above it. I think what you commented above is right.

Reply:
Oh -- actually the comment is wrong. Removing.

Reply:
Since users don't frequently create scenes using the API, we're going to leave this as is for now.

In the upload processing case, the scene is a COG and gets created with a status of `INGESTED` and a pointer to where that COG lives. In the ingest case, we handle the status cases in the backend. In the pointer to an existing COG case, we create the scene with a status of ingested and the COG location. This won't come up until we have users programmatically creating scenes, which we'll deal with when that starts to happen.